### PR TITLE
ConnectFour

### PR DESCRIPTION
**Pull request by Shauryaditya**
This is a simple game which can be played between two players . The player who connects four bubbles first wins the game . The bubbles could be joined horizontally , vertically or diagonally.

### CODE REVIEW

**Comment by Shauryaditya** [comment]
Please review my Pull Request.

**Comment by chicken-biryani** [comment]
Add your project Title and description in index.md

File: ConnectFour/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Technologies Used :
+
+### HTML
+
+### CSS
+
+### Javascript
+
+### JQuery
+
```
Comment (chicken-biryani) [inline ConnectFour/README.md:16]: Remove bold and put in just in bullet points.

File: ConnectFour/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+## How to set-up:
+
+### Clone this repo
+
+### Open it in any Text Editor
+
+### Enjoy it !!!
```
Comment (chicken-biryani) [inline ConnectFour/README.md:23]: Remove bold. Add clone command.

File: ConnectFour/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Connect Four
+
```
Comment (chicken-biryani) [inline ConnectFour/README.md:2]: Add description and use of project section.

File: ConnectFour/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+![ConnectFour](https://user-images.githubusercontent.com/68684482/124650666-98456c00-deb7-11eb-80e7-87e999d8ab96.png)
+
+### This is a simple game which can be played between two players.The player who connects the four bubbles first wins the game . The bubbles could be connected horizontally , vertically or diagonally.The players have to be together since multiplayer feature is not available .
```
Comment (chicken-biryani) [inline ConnectFour/README.md:5]: Remove bold

File: ConnectFour/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Connect Four
+
+![ConnectFour](https://user-images.githubusercontent.com/68684482/124650666-98456c00-deb7-11eb-80e7-87e999d8ab96.png)
+
```
Comment (chicken-biryani) [inline ConnectFour/README.md:4]: Put image in end.

File: ConnectFour/Part3_FrontEndProject.html [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    <script
+    src="https://code.jquery.com/jquery-3.1.1.min.js"
+    integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+    crossorigin="anonymous">
+    </script>
+    <script src='Part3_FrontEndProject.js'>
+
+    </script>
+  </body>
+</html>
```
Comment (chicken-biryani) [inline ConnectFour/Part3_FrontEndProject.html:86]: Add new line at end of file.

Use Html beautifier

File: ConnectFour/Part3_FrontEndProject.js [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+    currentName = player1;
+    $('h3').text(currentName+": it is your turn, please pick a column to drop your blue chip.");
+    currentColor = player1Color;
+  }else {
+    currentName = player2
+    $('h3').text(currentName+": it is your turn, please pick a column to drop your red chip.");
+    currentColor = player2Color;
+  }
+
+})
```
Comment (chicken-biryani) [inline ConnectFour/Part3_FrontEndProject.js:142]: Add a new line at the end of the file.

Use JS beautifier.

Comment (chicken-biryani) [inline ConnectFour/Part3_FrontEndProject.js:142]: Add a new line at the end of the file.

Use JS beautifier.

File: ConnectFour/Part3_FrontEndProject.css [hunk 8]
```diff
@@ -0,0 +1,8 @@
+.board button {
+  width: 100px;
+  height: 100px;
+  background-color: gray;
+  border-radius: 50%;
+  border: 4px solid black; /*this gets rid of bootstrap shadow*/
+  margin: 1px;
+}
```
Comment (chicken-biryani) [inline ConnectFour/Part3_FrontEndProject.css:8]: Use CSS beautifier.

**Review by chicken-biryani — COMMENTED** [review]
- Add project name in form of link and description in index.md
- Please use the following tools to increase the readability of your README / Scripts before creating a pull request.

It is a good practice.

1) Markdown - https://codebeautify.org/markdown-formatter
2) HTML - https://codebeautify.org/htmlviewer
3) CSS - https://codebeautify.org/css-beautify-minify
4) JS -  https://codebeautify.org/jsviewer

Use this PR as a reference :
https://github.com/Ayushparikh-code/Web-dev-mini-projects/pull/93

File: ConnectFour/Part3_FrontEndProject.css (hunk 8 above)
Comment (Shauryaditya) [reply ConnectFour/Part3_FrontEndProject.css:8]: @chicken-biryani please review the changes.

**Review by harshita2216 — APPROVED** [review]

**Review by Ayushparikh-code — APPROVED** [review]

**Comment by chicken-biryani** [comment]
@Shauryaditya add your project's name and description to index.md

File: ConnectFour/Part3_FrontEndProject.html (hunk 6 above)
Comment (chicken-biryani) [reply ConnectFour/Part3_FrontEndProject.html:86]: Add a new line at the end of file.

File: ConnectFour/README.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+# Connect Four
+
+- This is a simple game which can be played between two players.The player who connects the four bubbles first wins the game . The bubbles could be connected horizontally , vertically or diagonally.The players have to be together since multiplayer feature is not available .
+
```
Comment (chicken-biryani) [inline ConnectFour/README.md:4]: Add a "Use of project" section

**Comment by Shauryaditya** [comment]
> @Shauryaditya add your project's name and description to index.md

It's done . Please review the changes.

**Review by Ayushparikh-code — APPROVED** [review]

**Pull request merged**